server/routes/post/userSettings.js

```diff
@@ -1,9 +1,18 @@
 const isImage = require('is-image')
 import validator from 'validator'
 
+import cityLocations from '../../data/cityLocations.json'
 import message from '../../data/messages.json'
 import User from '../../database/models/user.js'
 
+function findExactCity(value) {
```

Comment:
Ik vind het zelf prima als functions zo geschreven worden, maar als we 1 ding aan moeten houden en daarop beoordeeld worden (weet ik niet zeker trouwens), dan misschien herschrijven naar `const findExactCity = (value) => { return ... }`

Laat me weten of dat de bedoeling is.